addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -71,11 +75,21 @@ export class WebsiteLoader extends Component {
             (isVisible) => {
                 if (isVisible) {
                     window.addEventListener("beforeunload", this.showRefreshConfirmation);
+                    if (!this.state.selectedFeatures || this.state.selectedFeatures.length === 0) {
+                        // If there is no feature selected, we fake the progress
+                        // for theme installation and configurator_apply. If
+                        // there is at least 1 feature selected, the progress
+                        // bar will be initialized in trackModules().
+                        this.initProgressBar();
+                    }
                 } else {
                     window.removeEventListener("beforeunload", this.showRefreshConfirmation);
                 }
 
-                return () => window.removeEventListener("beforeunload", this.showRefreshConfirmation);
+                return () => {
+                    window.removeEventListener("beforeunload", this.showRefreshConfirmation);
+                    clearInterval(this.updateProgressInterval);
+                }
```

Comment:
You have a problem with your linter :grimacing: Missing `;` here :grimacing:

Reply:
Yes I will check that